client/components/product-expiration/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+interface Moment {
+	isValid: () => boolean;
+	format: ( arg0: string ) => string;
+	diff: ( arg0: Date ) => number;
+}
+
+export interface Props {
+	expiryDateMoment: Moment;
+	purchaseDateMoment: Moment;
+	isRefundable: boolean;
+	dateFormat: string;
+	translate: ( arg0: string, arg1: object ) => string;
+}
+
+class ProductExpiration extends React.PureComponent< Props > {
+	static defaultProps = {
+		expiryDateMoment: null,
+		purchaseDateMoment: null,
+		isRefundable: false,
+		dateFormat: 'LL',
+	};
+
+	render() {
+		const {
+			expiryDateMoment,
+			purchaseDateMoment,
+			isRefundable,
+			dateFormat,
```

Comment:
Nit: wanna alphabetize these props?

Reply:
Resolved in 16feb54d14598303f399cc90336be44c634d157b